### ISSUE TITLE
about fca_layer.py

### ISSUE THREAD

**Issue by 123wk45678** [PlumedSerpent/FcaNet#3]
我发现你那个fca_layer.py 里面的FcaLayer，在forward里面，有一段这样的代码：
y = F.adaptive_avg_pool2d(x,(self.height,self.width))
y = torch.sum(x*self.pre_computed_dct_weights,dim=(2,3))
y = self.fc(y).view(b, c, 1, 1)
我发现这个"y = F.adaptive_avg_pool2d(x,(self.height,self.width))"在后面好像没有调用吧？

**Comment by PlumedSerpent** [comment]
Sorry for that bug. The results are obtained under the correct implementation.

**Comment by 123wk45678** [comment]
也就是说应该是这样的吗：
y = F.adaptive_avg_pool2d(x,(self.height,self.width))
y = torch.sum(y*self.pre_computed_dct_weights,dim=(2,3))
y = self.fc(y).view(b, c, 1, 1)

**Comment by PlumedSerpent** [comment]
Yes. I've updated this line. Thanks.
